<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ram-PC" failures="2" tests="2" name="Sanity in Firefox" time="268.195" errors="0" timestamp="2023-03-29T18:42:19 IST">
  <testcase classname="cse_testscripts.LoginFunctionality" name="checkLoginFucntionality" time="252.728">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at cse_testscripts.LoginFunctionality.checkLoginFucntionality(LoginFunctionality.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- checkLoginFucntionality -->
  <testcase classname="cse_testscripts.LoginFunctionality" name="checkLoginFucntionality" time="261.105">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at cse_testscripts.LoginFunctionality.checkLoginFucntionality(LoginFunctionality.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- checkLoginFucntionality -->
</testsuite> <!-- Sanity in Firefox -->
